### DISCUSSION TITLE
Can I make parallel requests?

### DISCUSSION THREAD

**Discussion by RenanLopes77** [Biscoint/biscoint-arbitrage-bot#20] [Q&A] [answered]
I was trying to make the buy and sell call together with "Promise.all", but it doesn't seem to be possible.

Error: `{"code":4003,"error":"Forbidden","details":"Invalid authorization headers"}`

**Comment by augustresende** [comment, marked as answer]
@RenanLopes77 it's not possible to use same api key to make parallel requests
